### PR TITLE
feat: return pr number, if one was opened

### PR DESCRIPTION
**Pull request by bcoe**
the PR number of an opened PR will now be returned, this allows us to run a GitHub action as a side effect of a PR being opened.

### CODE REVIEW

**Review by sofisl — APPROVED** [review]

File: src/release-pr.ts [hunk 1]
```diff
@@ -128,29 +128,31 @@ export class ReleasePR {
...
         'snapshot releases not supported for this releaser',
         CheckpointType.Failure
       );
       return;
     }
-    const pr: GitHubReleasePR | undefined = await this.gh.findMergedReleasePR(
-      this.labels
-    );
-    if (pr) {
+    const mergedPR:
```
Comment (JustinBeckwith) [inline src/release-pr.ts:139]: No need to be explicit with the type declaration here

File: src/release-pr.ts [hunk 2]
```diff
@@ -128,29 +128,31 @@ export class ReleasePR {
...
       checkpoint(
-        `pull #${pr.number} ${pr.sha} has not yet been released`,
+        `pull #${mergedPR.number} ${mergedPR.sha} has not yet been released`,
         CheckpointType.Failure
       );
+      return undefined;
     } else {
-      return this._run();
+      const openedPR = this._run();
+      return openedPR ? openedPR : undefined;
```
Comment (JustinBeckwith) [inline src/release-pr.ts:151]: Huh - is this trying to normalize null and undefined?

**Review by JustinBeckwith — APPROVED** [review]

File: src/release-pr.ts (hunk 2 above)
Comment (bcoe) [reply src/release-pr.ts:151]: we return `0` from our github helper, I figured for the benefit of the GitHub action `undefined` would be a better signal that we didn't do anything.

I suppose we could switch to `undefined` everywhere.

Comment (bcoe) [reply src/release-pr.ts]: switched to returning `undefined` everywhere.

**Pull request merged**